paddle/phi/api/yaml/op_compat.yaml

```
@@ -2653,6 +2653,14 @@
   outputs :
     out : Out
 
+- op : pull_sparse_v2
+  inputs :
+    { ids : Ids, W : w}
```

```suggestion
    { ids : Ids, w : W}
```
push_sparse_v2也需要同步修改一下。